### PR TITLE
Integrations/jaeger

### PR DESCRIPTION
**Pull request by imnotashrimp**
# What changed

- Drafted Jaeger instructions
- moved Zipkin to Integrations section in the TOC (was previously in the shipping page)

----

## Pages to review

- https://deploy-preview-355--logz-docs.netlify.com/integrations/store-jaeger-traces.html

## Remaining work

- [x] Technical review
- [x] Copy Review
- [x] Redirects: /shipping/log-sources/zipkin.html

## Post launch

To be completed by the docs team upon merge:

- [ ] push the GFM version of Zipkin readme
- [ ] remove Ziipkin from log shipping pages in the app

### CODE REVIEW

File: _source/integrations/jaeger.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+logzio/jaeger-logzio:latest
+```
+
+###### Environment variables
+
+| Parameter | Description |
+|---|---|
+| ACCOUNT_TOKEN <span class="required-param"></span> | Required when using as a collector to ship traces to Logz.io. <br> {% include log-shipping/replace-vars.html token=true %} |
+| API_TOKEN	<span class="required-param"></span> | Required to read back traces from Logz.io. <br> Replace `<<API-TOKEN>>` with an [API token](https://app.logz.io/#/dashboard/settings/api-tokens) from the account you want to use. |
+| REGION <span class="default-param">`us`</span> | Two-letter region code. This determnies your listener URL (where you're shipping the logs to) and API URL. <br> Replace `us` with your region's code. |
```
Comment (yyyogev) [inline _source/integrations/jaeger.md]: Shouldn't we add a link to the account region?

File: _source/integrations/jaeger.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+  -p 14269:14269 \
+  -p 14250:14250 \
+logzio/jaeger-logzio-collector:latest
+```
+
+###### Environment variables
+
+| Parameter | Description |
+|---|---|
+| ACCOUNT_TOKEN <span class="required-param"></span> | Required when using as a collector to ship traces to Logz.io. <br> {% include log-shipping/replace-vars.html token=true %} |
```
Comment (yyyogev) [inline _source/integrations/jaeger.md]:
```suggestion
| ACCOUNT_TOKEN <span class="required-param"></span> | <br> {% include log-shipping/replace-vars.html token=true %} |
```
This is a collector only..

File: _source/integrations/jaeger.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+  -p 16687:16687 \
+  --name=jaeger-logzio-query \
+logzio/jaeger-logzio-query:latest
+```
+
+###### Environment variables
+
+| Parameter | Description |
+|---|---|
+| API_TOKEN	<span class="required-param"></span> | Required to read back traces from Logz.io. <br> Replace `<<API-TOKEN>>` with an [API token](https://app.logz.io/#/dashboard/settings/api-tokens) from the account you want to use. |
```
Comment (yyyogev) [inline _source/integrations/jaeger.md]:
```suggestion
| API_TOKEN	<span class="required-param"></span> |  <br> Replace `<<API-TOKEN>>` with an [API token](https://app.logz.io/#/dashboard/settings/api-tokens) from the account you want to use. |
```

same

File: _source/integrations/jaeger.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+start it up with this command:
+
+```shell
+docker run --rm --name=jaeger-agent --network=net-logzio \
+  -p5775:5775/udp \
+  -p6831:6831/udp \
+  -p6832:6832/udp \
+  -p5778:5778/tcp \
+  jaegertracing/jaeger-agent:latest \
+  --reporter.tchannel.host-port=jaeger-logzio:14267
```
Comment (yyyogev) [inline _source/integrations/jaeger.md]:
```suggestion
  --reporter.tchannel.host-port=jaeger-logzio-collector:14267
```
Match the name above

File: _source/integrations/jaeger.md (hunk 1 above)
Comment (yyyogev) [inline _source/integrations/jaeger.md]: Add a link to account region?

Comment (imnotashrimp) [reply _source/integrations/jaeger.md]: Added.

**Pull request merged**